prow/plugins/trigger/ic.go

```diff
@@ -54,13 +54,33 @@ func handleIC(c client, trustedOrg string, ic github.IssueCommentEvent) error {
 		return nil
 	}
 
+	var changedFiles []string
+	files := func() ([]string, error) {
+		if changedFiles != nil {
+			return changedFiles, nil
+		}
+		changes, err := c.GitHubClient.GetPullRequestChanges(org, repo, number)
+		if err != nil {
+			return nil, err
+		}
+		changedFiles = []string{}
```

Comment:
`append` should work with `nil`.

Reply:
it does, but it seems cleaner to return an empty slice here instead of a nil slice

Reply:
I think the opposite but [1] describes it better. Anyway, this is more of a convention.

[1] https://softwareengineering.stackexchange.com/a/120356

Reply:
I set it to an empty slice to differentiate the case where there are no PR changes from the case where we haven't yet queried for PR changes.
I can add a bool to distinguish these cases if that is preferable to setting `changedFiles` to an empty slice.

Reply:
No, that's fine.